docs/ui-ux/README.md

```diff
@@ -0,0 +1,80 @@
+
+User stories help you and your team understand application and feature requirements. In application design, the user drives the features. By completely understanding your user, you in turn, can completely understand what you need to build.
+
+Following the user stories, the design document should detail the accessibility requirements of the application. **Accessibility is never optional**. Microsoft has made a public commitment to deliver No matter who you determine your users are, there is a strict set of minimum requirements for all [Microsoft applications](https://www.microsoft.com/en-us/accessibility).
+
+> If you haven't already, download the [Accessibility Insights](https://accessibilityinsights.io/) browser tool. You can use this, in combination with other accessibility-related tools, to easily check your application.
+
+Accessibility requirements vary from project-to-project based on your intended user base. Aside from the default level of accessibility, this section should contain details about any other aspects you must consider during development. Could the application be used by someone who cannot see? What about someone with color blindness? What about someone who cannot use a mouse? If your application has an audio aspect, what about users that cannot hear?
+
+> Did you know that approximately 4.5% of the worlds population is color blind[*](https://www.colorblindguide.com/post/colorblind-people-population-live-counter)? That is over 355 Million people! Accessible design is a requirement for all user interface applications.
+
+After listing accessibility requirements, the last section of the design document is technical considerations. This section outlines any and all implementation options your team wants to consider. Within CSE, we often reach for tools such as the [React] framework. React is a great tool when wielded by an experienced team. Otherwise, it can create more hurdles than it is worth. Keep in mind that even if _you_ feel capable with React, the rest of your team and your customer's dev team needs to as well. Some other great options to consider when building a proof-of-concept or minimum-valuable-product are:
+
+1. HTML/CSS/JavaScript
+   - Back to the basics! Start with a single **index.html**, include a popular CSS framework such as [Bootstrap]() using their CDN link, and start prototyping!
+   - Rarely will you have to support legacy browsers; thus, you can rely on modern JavaScript language features! No need for build tools or even TypeScript (did you know you can [type check JavaScript]()?)
```

Comment:
Missing links here, maybe part of the WIP 🙂

Reply:
I may suggest rewording this point, for example:

- Typically, the default output for client-side JavaScript is sufficient for an MVP app, if running in production and targetting legacy browsers (which happens but is more uncommon) then look into polyfills and configuration such as (browserlist)[https://github.com/browserslist/browserslist]. 
- Typescript is recommended for new projects however JavaScript can be sufficient as well, but it lacks the robust Typing system of Typescript.